app/scripts/controllers/groups/TransferClientsController.js

```diff
@@ -44,8 +60,8 @@
                 }
             };
             
-            scope.viewgroup = function (id) {
-                resourceFactory.groupResource.get({groupId: id, associations: 'all'}, function (data) {
+            scope.viewgroup = function (group) {
+                resourceFactory.groupResource.get({groupId: group.id, associations: 'all'}, function (data) {
```

Comment:
do we need "all" association here ? If we don't, we could remove the same to improve performance